source/server/server.h

```diff
@@ -74,6 +74,7 @@ namespace Server {
   GAUGE(total_connections, Accumulate)                                                             \
   GAUGE(uptime, Accumulate)                                                                        \
   GAUGE(version, NeverImport)                                                                      \
+  GAUGE(fips_mode, NeverImport)                                                                    \
```

Comment:
In order to add similar stats for other compile-time settings in the future, please move this into a nested namespace, so the full name would be `server.compilation_settings.fips_mode`.

Reply:
Moved it to a nested namespace